docs/index.md

```diff
@@ -94,7 +99,7 @@ There are now various different ways to let the Python environment know of these
 
 #### `setup.py`
 
-In the package's `setup.py` file, one can specify entry points.  
+In the package's `setup.py` file, one can specify entry points.
```

Comment:
```suggestion
In the package's `setup.py` file, one can specify entry points.  
```
Adding this change manually here as well relinquishes the need for running `pre-commit` again locally and this can be accepted and merged.